web/annotation_layer_builder.css

```diff
@@ -93,6 +93,64 @@
   width: 115%;
 }
 
+.annotationLayer .checkboxWidgetAnnotation label,
+.annotationLayer .radioButtonWidgetAnnotation label {
+  background-color: rgb(222, 229, 255);
```

Comment:
I cannot reproduce this, please provide a PDF file that demonstrates this problem!
Without having seen the file, it sounds like it may be an issue related to `Appearance` streams support in conjunction with forms, which is a somewhat difficult issue that we'll have to somehow fix before we can enable forms by default.

However, simply hiding the background in this way isn't going to be a working solution, so please change this for now.